.github/workflows/test.yaml

```diff
@@ -37,11 +37,37 @@ jobs:
           git diff --exit-code
 
       - name: Run tests with coverage
-        run: go test ./... --short --race --covermode=atomic --coverprofile=coverage.out
+        run: go test $(go list ./... | grep -v /test) --short --race --covermode=atomic --coverprofile=coverage.out
```

Comment:
wouldn't it be better to refer to cucumber specifically here (to avoid possible addition of tests in future)?

```sh
grep -v cucumber
```